samples/core/kfp_env_validation/kfp_env_validation.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>In line 8 the base image should be <code>google/cloud-sdk:276.0.0</code> read from below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done :)